public/static/less/host.less

```diff
@@ -11,6 +11,17 @@
   background-color: #5F923B;
   color: white;
 }
+.host-name-input {
+  border: none;
+  width: 100%;
+}
+.host-name-input:focus {
+  background-color: #9FBD89;
+}
+.host-name-confirm {
+  position: relative;
```

Comment:
You might have tried this already, but would it be possible to keep the buttons inside their boxes? Right now the name can bleed into the buttons (this is full screen on my 15" laptop)

<img width="858" alt="Screen Shot 2020-03-30 at 12 03 19 PM" src="https://user-images.githubusercontent.com/29384396/77934883-b2d57200-727e-11ea-9328-43149c1bf2e5.png">

Reply:
I thought to leave the buttons inside the input box (so it's clear they go together with the input and they don't take up extra space when they're not visible), but I added 50% opacity to the buttons and made the color darker. What do you think?

<img width="601" alt="Screen Shot 2020-03-31 at 12 36 06 PM" src="https://user-images.githubusercontent.com/17027265/78051997-7e2fec00-734c-11ea-8148-4967f3496774.png">

Reply:
What if you split it into 2 boxes, 1 that is a text input and one to contain both the input and buttons? That way you can style the outer so that it visually groups the input and buttons

Reply:
done.
 It's much better now. thanks!

<img width="289" alt="Screen Shot 2020-03-31 at 2 49 24 PM" src="https://user-images.githubusercontent.com/17027265/78064107-3bc3da80-735f-11ea-9b00-d982cba40b80.png">